src/lib/components/BodyRenderer/Blocks/testimonial.svelte

```diff
@@ -1,76 +1,39 @@
 <script lang="ts">
   import type { TestimonialElement } from '$components/BodyParser/blocks';
-  import { getContext } from 'svelte';
-  import type { Writable } from 'svelte/store';
-  /* eslint-disable @typescript-eslint/no-unused-vars */
-
+  import Link from '$components/BodyRenderer/Blocks/link.svelte';
   export let item: TestimonialElement;
 
-  let themeContext: Writable<string> = getContext('theme');
+  const picture =
+    item.picture?.url && item.picture.url.trim() !== ''
+      ? item.picture.url
+      : '/empty-image-thumb.png';
 </script>
 
 <div
-  class={'border-decoration relative flex-col justify-start items-stretch w-full border border-solid border-l4 rounded-xl bg-l1 dark:bg-l2 border-decoration-' +
-    $themeContext}
+  class="testimonial-wrapper flex flex-col gap-4 w-full my-12 items-center smDown:items-start xl-up:py-10 xl-up:px-28 tb:p-20 pt-8 smDown:px-4"
 >
-  <div class="flex flex-row border-b border-solid border-l4 p-4">
-    <div class="flex items-center">
-      {#if item.picture}
-        <img class="w-12 h-12 rounded-full object-cover object-center" src={item.picture.url} />
-      {/if}
+  <div class="testimonial relative">
+    <span
+      class="absolute text-[201px] font-satoshi text-t3 -top-36 -left-25 tb:-left-2 tb:-top-24 tb:text-9xl sm:text-9xl sm:-top-20 sm:left-1 xs:text-9xl xs:-top-18 xs:left-1"
+      >“</span
+    >
+    <div class="testimonial-text">
+      <blockquote class="text-paragraph-l text-t2 xs:text-paragraph-s max-w-[604px]">
+        {@html item.content}
+      </blockquote>
     </div>
-    <div class="flex flex-col xs:w-full ml-4">
-      {#if item.name}
-        <p class="text-paragraph-l text-t1 font-bold">{item.name}</p>
-      {/if}
-
-      {#if item.title}
-        <p class="text-paragraph-l text-t3">{item.title}</p>
-      {/if}
+    <div class="author-info flex items-center justify-end gap-4 mt-4">
+      <div
+        class="author-details flex flex-col items-end text-right font-satoshi text-h5-l smDown:text-h5-s"
+      >
+        <svelte:component
```

Comment:
Why not just 
```suggestion
        <Link ....
```

?

Reply:
You're right, better to call it directly. Fixed it!